File: spring-armeria-reactor/src/main/java/com/example/ArmeriaServerConfig.java
package com.example;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import com.linecorp.armeria.server.logging.AccessLogWriter;
import com.linecorp.armeria.spring.ArmeriaServerConfigurator;

@Configuration
public class ArmeriaServerConfig {
    @Bean
    public ArmeriaServerConfigurator armeriaServerConfigurator(ReactorService reactorService) {
        return builder -> builder.annotatedService(reactorService)
                                 .accessLogWriter(AccessLogWriter.combined(), false);
    }
}

File: spring-boot-kotlin/src/main/kotlin/com/example/ArmeriaServerConfig.kt
package com.example

import com.linecorp.armeria.server.logging.AccessLogWriter
import com.linecorp.armeria.spring.ArmeriaServerConfigurator
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class ArmeriaServerConfig {
    @Bean
    fun armeriaServerConfigurator(helloService: HelloService): ArmeriaServerConfigurator =
        ArmeriaServerConfigurator { builder ->
            builder.annotatedService(helloService)
                .accessLogWriter(AccessLogWriter.combined(), false)
        }
}
